public/.eslintcache

```diff
@@ -0,0 +1 @@
+[{"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\index.js":"1","D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\reportWebVitals.js":"2","D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\App.js":"3","D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\header.js":"4","D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\footer.js":"5","D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\newsletter.js":"6","D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\recently_added.js":"7"},{"size":717,"mtime":1617397632603,"results":"8","hashOfConfig":"9"},{"size":375,"mtime":1617228843781,"results":"10","hashOfConfig":"9"},{"size":1903,"mtime":1617403664325,"results":"11","hashOfConfig":"9"},{"size":204,"mtime":1617228843782,"results":"12","hashOfConfig":"9"},{"size":3163,"mtime":1617396136201,"results":"13","hashOfConfig":"9"},{"size":2364,"mtime":1617395854098,"results":"14","hashOfConfig":"9"},{"size":3371,"mtime":1617470282140,"results":"15","hashOfConfig":"9"},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},"oa5tie",{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"27","messages":"28","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"29","messages":"30","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\index.js",[],["31","32"],"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\reportWebVitals.js",[],"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\App.js",[],"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\header.js",[],"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\footer.js",[],"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\newsletter.js",[],"D:\\Programming\\TBDG\\ofc-front-end\\public\\src\\shared\\recently_added.js",[],{"ruleId":"33","replacedBy":"34"},{"ruleId":"35","replacedBy":"36"},"no-native-reassign",["37"],"no-negated-in-lhs",["38"],"no-global-assign","no-unsafe-negation"]
```

Comment:
again we shouldn't be checking in this code

Reply:
wtf i'm not. it's in my git ignore?! why isn't that working?

Reply:
You will probably need to do something like we did last time.

1. take it out of .gitignore
2. commit 
3. delete the file 
4. commit
5. add it to .gitignore
6. commit

 take it out of the git ignore